server/api/controllers/forms/get-form.js

```diff
@@ -15,10 +15,26 @@ module.exports = {
 
   fn: async function ({ id }, exits) {
     try {
+      let now = new Date();
+      let offset = now.getTimezoneOffset();
+      now.setTime(now.getTime() - offset * 60000);
+      now = now.toISOString();
+
```

Comment:
imo we can move this to a separate helper function, may need to use it again in the future